test/models/testdb/model with spaces/meshes/material with spaces.mtl

```diff
@@ -1,8 +1,7 @@
-newmtl everything is ignored after first space
+newmtl MaterialNameWithoutSpaces
   Ns 10.0
   Ni 1.5
   d 1.0
-  Tr 0.0
```

Comment:
Keeping this here now gives warnings that "Both `d` and `Tr` parameters defined ... Use the value of `d` for dissolve". That was changed since version 1.0.5 and refers to this issue: https://github.com/tinyobjloader/tinyobjloader/issues/43.